components/clp-package-utils/clp_package_utils/scripts/compress.py

```diff
@@ -141,7 +141,9 @@ def main(argv):
     args_parser.add_argument(
         "--no-progress-reporting", action="store_true", help="Disables progress reporting."
     )
-    args_parser.add_argument("paths", metavar="PATH", nargs="*", help="Paths to compress.")
+    args_parser.add_argument(
+        "paths", metavar="PATH", nargs="*", help="Paths or S3 URL to compress."
```

Comment:
maybe we should add "an"?
```suggestion
        "paths", metavar="PATH", nargs="*", help="Paths or an S3 URL to compress."
```